rest-api/api/awardee_api.py

```diff
@@ -0,0 +1,24 @@
+from api.base_api import BaseApi
+from app_util import auth_required
+from api_util import PTC_AND_HEALTHPRO
+from dao.hpo_dao import HPODao
+from werkzeug.exceptions import NotFound
+
+class AwardeeApi(BaseApi):
+  def __init__(self):
+    super(AwardeeApi, self).__init__(HPODao(), get_returns_children=True)
+
+  @auth_required(PTC_AND_HEALTHPRO)
+  def get(self, a_id=None):
+    if a_id:
+      hpo = self.dao.get_by_name(a_id)
+      if not hpo:
+        raise NotFound("Awardee with ID %s not found" % a_id)
+      return self._make_response(self.dao.get_with_children(hpo.hpoId))
+    else:
+      return super(AwardeeApi, self)._query(id_field='id')
+
+  def _make_resource_url(self, json, id_field, participant_id):
+    import main
```

Comment:
This is pretty sketchy.. but I guess it's an existing issue. I was going to suggest the following approach, but it would now probably be a larger refactoring since every API class depends on some form of this:

Accept `api` as a param to the `AwardeeApi` ctor, then pass `api` via add_resource using resource_class_args? https://flask-restful.readthedocs.io/en/latest/api.html#flask_restful.Api.add_resource

Reply:
Yeah, we could do that. I'm not sure I like it much better than this. :)

For now, will leave it as-is.